Basic-Recruit-Training/main.cpp

```diff
@@ -19,22 +19,30 @@ void compareAnswers(const std::string myAnswer, const std::string correctAnswer)
 // Swaps a & b.
 void mySwap(int& a, int& b)
 {
+    int temp = a;
+    a = b;
+    b = temp;
 }
 // Increment the int that x is pointing to by one.
 void myIncrement(int* x)
 {
+    ++*x;
```

Comment:
A good idea when using multiple non-mathematical operators like the one here, is using brackets to differentiate which operation comes first